packages/dev/serializers/src/USD/USDSerializer.ts

```diff
@@ -0,0 +1,694 @@
+            inputs.push(
+                `${pad}color3f inputs:emissiveColor.connect = </Materials/Material_${material.uniqueId}/Texture_${material.emissiveTexture.uniqueId}_emissive.outputs:rgb>`
+            );
+            samplers.push(buildTexture(material.emissiveTexture as Texture, "emissive"));
+        } else if (material.emissiveColor) {
+            inputs.push(`${pad}color3f inputs:emissiveColor = ${USDExport._BuildColor(material.emissiveColor)}`);
+        }
+
+        if (material.bumpTexture !== null) {
+            inputs.push(`${pad}normal3f inputs:normal.connect = </Materials/Material_${material.uniqueId}/Texture_${material.bumpTexture.uniqueId}_normal.outputs:rgb>`);
+            samplers.push(buildTexture(material.bumpTexture as Texture, "normal"));
+        }
+
+        // if (material.ambientTexture !== null) {
+        //   inputs.push(
+        //     `${pad}float inputs:occlusion.connect = </Materials/Material_${material.uniqueId}/Texture_${material.ambientTexture.uniqueId}_occlusion.outputs:r>`
+        //   );
+        //   samplers.push(
+        //     buildTexture(material.ambientTexture as Texture, "occlusion")
+        //   );
+        // }
+
+        if (material.metallicTexture !== null) {
+            const maps = await breakApartMetallicRoughnessAo(material.metallicTexture as Texture);
+
+            inputs.push(`${pad}float inputs:roughness.connect = </Materials/Material_${material.uniqueId}/Texture_${maps.roughness.uniqueId}_roughness.outputs:g>`);
+            samplers.push(buildTexture(maps.roughness, "roughness"));
+
+            inputs.push(`${pad}float inputs:metallic.connect = </Materials/Material_${material.uniqueId}/Texture_${maps.metallic.uniqueId}_metallic.outputs:b>`);
+            samplers.push(buildTexture(maps.metallic, "metallic"));
+
+            inputs.push(`${pad}float inputs:occlusion.connect = </Materials/Material_${material.uniqueId}/Texture_${maps.ao.uniqueId}_occlusion.outputs:r>`);
+            samplers.push(buildTexture(maps.ao, "occlusion"));
+        } else {
+            inputs.push(`${pad}float inputs:roughness = ${material.roughness}`);
+            inputs.push(`${pad}float inputs:metallic = ${material.metallic}`);
+        }
+
+        if (material.opacityTexture !== null) {
+            inputs.push(`${pad}float inputs:opacity.connect = </Materials/Material_${material.uniqueId}/Texture_${material.opacityTexture.uniqueId}_opacity.outputs:r>`);
+            inputs.push(`${pad}float inputs:opacityThreshold = 0.0001`);
+            samplers.push(buildTexture(material.opacityTexture as Texture, "opacity"));
+        } else {
+            inputs.push(`${pad}float inputs:opacity = ${material.alpha}`);
+        }
+
+        if (material.isMetallicWorkflow()) {
+            inputs.push(`${pad}float inputs:clearcoat = ${0.0}`);
+            inputs.push(`${pad}float inputs:clearcoatRoughness = ${0.1}`);
+            inputs.push(`${pad}float inputs:ior = ${material.metallicF0Factor}`);

```

Comment:
It's probably more `ior = material.subSurface?.indexOfRefraction ?? 1.5;` than `metallicF0Factor` here.

See:
 https://github.com/BabylonJS/Babylon.js/blob/master/packages/dev/core/src/Materials/PBR/pbrBaseMaterial.ts#L2175-L2184